tests/ui/associated-inherent-types/private-in-public.rs

```diff
@@ -2,22 +2,14 @@
 #![feature(type_privacy_lints)]
 #![allow(incomplete_features)]
 #![crate_type = "lib"]
-#![deny(private_in_public)]
-#![warn(private_interfaces)]
-
-// In this test both old and new private-in-public diagnostic were emitted.
-// Old diagnostic will be deleted soon.
-// See https://rust-lang.github.io/rfcs/2145-type-privacy.html.
+#![deny(private_interfaces)]
```

Comment:
Could you set the default level for `private_interfaces` and `private_bounds` lints to warn, and remove all the extra annotations like this from tests?